### ISSUE TITLE
Do not colorise (or colorise properly) ESIL output

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#3943]
In mode `e asm.esil=true`.
![asm_esil](https://cloud.githubusercontent.com/assets/203261/12231376/047c6b48-b868-11e5-9799-f0beb0047b1a.png)

**Comment by radare** [comment]
? it is colorized depending on the type of instruction. so that scorrect

**Comment by XVilka** [comment]
Yes, I got it. But may be it is better not to colorize it such way? Just discussing.

**Comment by radare** [comment]
we will need to write a new colorization for esil expressions, but the idea looks good to me. having optype color looks also useful.. so what do you think?

**Comment by XVilka** [comment]
yep, like switching from optype colorization to nothing or to ESIL-specific highlighting on the fly.

**Comment by trufae** [comment]
ESIL is not colorized right now, when you did the screenshot it was honoring the opcode type to pick the color. What do you exactly mean by "colorise properly"?

**Comment by wargio** [comment]
i think what he meant is like what we do for asm but on esil. probably change color of numbers, commas, etc..
as you can see from the screenshot some esil code is in orange, some other in green and most is in white.
